src/sentry/static/sentry/app/views/organizationIntegrations/installedPlugin.tsx

```diff
@@ -0,0 +1,164 @@
+import {Client} from 'app/api';
+import {
+  addErrorMessage,
+  addSuccessMessage,
+  addLoadingMessage,
+} from 'app/actionCreators/indicator';
+import {PluginNoProject, PluginProjectItem, Organization, AvatarProject} from 'app/types';
+
+export type Props = {
+  api: Client;
+  plugin: PluginNoProject;
+  projectItem: PluginProjectItem;
+  organization: Organization;
+  onResetConfiguration: (projectId: string) => void;
+  onEnablePlugin: (projectId: string) => void;
+  className?: string;
+};
+
+export class InstalledPlugin extends React.Component<Props> {
+  getConfirmMessage() {
+    return (
+      <React.Fragment>
+        <Alert type="error" icon="icon-circle-exclamation">
+          {t(
+            'Deleting this installation will disable the integration for this project and remove any configurations.'
+          )}
+        </Alert>
+      </React.Fragment>
+    );
+  }
+
+  pluginUpdate = async (data: object) => {
+    const {organization, projectItem, plugin} = this.props;
+    await this.props.api.requestPromise(
+      `/projects/${organization.slug}/${projectItem.projectSlug}/plugins/${plugin.id}/`,
+      {
+        method: 'POST',
+        data,
+      }
+    );
+  };
+
+  handleReset = async () => {
+    try {
+      addLoadingMessage(t('Removing...'));
+      await this.pluginUpdate({reset: true});
+      addSuccessMessage(t('Configuration was removed'));
+      this.props.onResetConfiguration(this.props.projectItem.projectId);
+    } catch (_err) {
+      addErrorMessage(t('Unable to remove configuration'));
+    }
+  };
+
+  handleUninstallClick = () => {
+    //TODO: Analytics
+  };
+
+  enablePlugin = async () => {
+    try {
+      addLoadingMessage(t('Enabling...'));
+      await this.pluginUpdate({enabled: true});
+      addSuccessMessage(t('Configuration was enabled'));
+      this.props.onEnablePlugin(this.props.projectItem.projectId);
+    } catch (_err) {
+      addErrorMessage(t('Unable to enable configuration'));
+    }
+  };
+
+  get projectForBadge(): AvatarProject {
+    //this function returns the project as needed for the ProjectBadge component
+    const {projectItem} = this.props;
+    return {
+      slug: projectItem.projectSlug,
+      platform: projectItem.projectPlatform,
+    };
+  }
+
+  render() {
+    const {className, plugin, organization, projectItem} = this.props;
+
+    return (
+      <Container>
+        <Access access={['org:integrations']}>
+          {({hasAccess}) => (
+            <IntegrationFlex className={className}>
+              <IntegrationItemBox>
+                <ProjectBadge project={this.projectForBadge} />
+              </IntegrationItemBox>
+              <div>
+                {!projectItem.enabled ? (
+                  <Button size="small" priority="primary" onClick={this.enablePlugin}>
+                    {t('Enable')}
+                  </Button>
+                ) : (
+                  <StyledButton
+                    borderless
```

Comment:
Why borderless in this context?

Reply:
@EvanPurkhiser It definitely doesn't look good here, that's for sure. We really just copy-pasted these buttons from the existing integrations page where it looks better:

![Screen Shot 2020-02-05 at 11 20 21 AM](https://user-images.githubusercontent.com/8533851/73875105-8ca8ce00-4809-11ea-957b-5fe82c23c09e.png)

Right now, we are waiting for a proper design from Chris. We'll tune up all the styling on this page once we know what it should look like.